src/http_server.cc

```diff
@@ -3327,6 +3327,7 @@ HTTPAPIServer::HandleGenerate(
   //   thus the string must live as long as the JSON message).
   triton::common::TritonJson::Value request;
   RETURN_AND_CALLBACK_IF_ERR(EVRequestToJson(req, &request), error_callback);
+  RETURN_AND_CALLBACK_IF_ERR(ParseJsonTritonRequestID(request, irequest), error_callback);
```

Comment:
Please fix the [pre-commit linter complaint](https://github.com/triton-inference-server/server/actions/runs/9857339520/job/27233990029):
```
-  RETURN_AND_CALLBACK_IF_ERR(ParseJsonTritonRequestID(request, irequest), error_callback);
+  RETURN_AND_CALLBACK_IF_ERR(
+      ParseJsonTritonRequestID(request, irequest), error_callback);
```

For future contribution reference, you can pre-emptively setup the pre-commit hooks before development `pre-commit install` or run them after the fact with `pre-commit run --all-files`: https://pre-commit.com/

Reply:
Fixed the pre-commit issue. Have set it up for future commits

Reply:
Thanks!